src/components/Calculator.js

```diff
@@ -1,43 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import '../index.css';
 import Buttons from './Buttons';
 import calculate from '../logic/calculate';
 
 console.log(React);
```

Comment:
[OPTIONAL] Kindly remove the `console.log` from your code to keep your code clean.